blazor/treegrid/events.md

```diff
@@ -2509,7 +2509,7 @@ The [DetailDataBound](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Sy
 
 ## RowDropped
 
-[RowDropped](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Syncfusion.Blazor.TreeGrid.TreeGridEvents%601~RowDropped.html) event triggers after the row elements are dragged and dropped on the TreeGrid. 
+[RowDropped](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.RowDroppedEventArgs-1.html) event triggers after the row elements are dragged and dropped on the TreeGrid. 
```

Comment:
Seems you refereed the Grid event URL here. Recheck and update TreeGrid Valid URL here.

Reply:
Updated